src/plugins/data/public/antlr/opensearch_ppl/opensearch_ppl_autocomplete.ts

```diff
@@ -31,6 +34,18 @@ export function getIgnoredTokens(): number[] {
     OpenSearchPPLParser.PIPE,
     OpenSearchPPLParser.EQUAL,
     OpenSearchPPLParser.COMMA,
+    OpenSearchPPLParser.PLUS,
+    OpenSearchPPLParser.MINUS,
+    // OpenSearchPPLParser.EQUAL,
+    // OpenSearchPPLParser.NOT_EQUAL,
+    // OpenSearchPPLParser.LESS,
+    // OpenSearchPPLParser.NOT_LESS,
+    // OpenSearchPPLParser.GREATER,
+    // OpenSearchPPLParser.NOT_GREATER,
+    // OpenSearchPPLParser.OR,
+    // OpenSearchPPLParser.AND,
+    // OpenSearchPPLParser.XOR,
+    // OpenSearchPPLParser.NOT,
```

Comment:
what does commenting them here mean?

Reply:
This is uncommented in a later PR, these keywords appear in places which might be grammatically correct but doesn't make sense for autocomplete, so in an effort to include these keywords I was working through them one at a time.